src/Controller/Api/DatasetSubmissionController.php

```diff
@@ -182,6 +183,9 @@ public function postAction(Request $request)
      *     )
      * )
      *
+     * @IsGranted("IS_AUTHENTICATED_FULLY")
+     * @IsGranted("CAN_EDIT", subject="datasetSubmission")
```

Comment:
I'm not sure if this `subject` is available, you may have to do it with that magic conversion method?

Reply:
Actually, don't need this line at all, entityHandler checks that.